toonz/sources/common/tgeometry/tgeometry.cpp

```diff
@@ -58,14 +58,18 @@ TAffine TAffine::inv() const {
   if (a12 == 0.0 && a21 == 0.0) {
     assert(a11 != 0.0);
     assert(a22 != 0.0);
```

Comment:
IMO these assertions should be removed since for now this function should work with the values `a11` and `a22` being 0.0.